addons/hr_homeworking/models/hr_employee.py

```diff
@@ -19,6 +19,10 @@ class HrEmployeeBase(models.AbstractModel):
     friday_location_id = fields.Many2one('hr.work.location', string='Friday')
     saturday_location_id = fields.Many2one('hr.work.location', string='Saturday')
     sunday_location_id = fields.Many2one('hr.work.location', string='Sunday')
+    exceptional_location_id = fields.Many2one(
+        'hr.work.location', string='Current',
```

Comment:
Though it won't be used since it's not in a view :D
```suggestion
        'hr.work.location', string='Exceptional Location',
```

Reply:
I will make the change when the task is in review ^^